llvm/test/tools/file-table-tform/file-table-tform-merge.test

```diff
@@ -0,0 +1,6 @@
+-- Merge two tables listed in merge-input.txt
+RUN: cp %S/Inputs/a{0,1}.txt .
```

Comment:
`.` is `build/test/tools/file-table-tform`

We already implicitly rely on `.` because we use `-o`, which obviously writes to `.`, so I think it's fine